apps/mobile/package.json

```diff
@@ -45,6 +45,7 @@
     "@leather.io/bitcoin": "workspace:*",
     "@leather.io/constants": "workspace:*",
     "@leather.io/crypto": "workspace:*",
+    "@leather.io/leather-provider": "workspace:*",
```

Comment:
Let's not call leather+leathe

Suggest just `provider` or `injected-provider`?